be/src/exprs/agg/percentile_approx.h

```diff
@@ -38,7 +39,33 @@ struct PercentileApproxState {
 
 class PercentileApproxAggregateFunction final
         : public AggregateFunctionBatchHelper<PercentileApproxState, PercentileApproxAggregateFunction> {
+private:
+    static constexpr double MIN_COMPRESSION = 2048.0;
+    static constexpr double MAX_COMPRESSION = 10000.0;
+    static constexpr double DEFAULT_COMPRESSION_FACTOR = 10000.0;
+
 public:
+    void create(FunctionContext* ctx, AggDataPtr __restrict ptr) const override {
```

Comment:
Can you give a default `compression` if `ctx` is null because `agg_state` in common aggregate state it may be null.

Reply:
Sure.